tests/views_integration/Tribe/Events/Views/V2/Partials/List_View/Top_Bar/DatepickerTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Tribe\Events\Views\V2\Partials\List_View\Top_Bar;
+
+use Tribe\Test\Products\WPBrowser\Views\V2\HtmlPartialTestCase;
+
+class DatepickerTest extends HtmlPartialTestCase
+{
+
+	protected $partial_path = 'list/top-bar/datepicker';
+
+	/**
+	 * Test render
+	 */
+	public function test_render() {
+		$this->markTestSkipped( 'The "today" variable is not working as expected' );
```

Comment:
Here and elsewhere: either in the code or in scrum report this to be addressed by someone.